src/flate.rs

```diff
@@ -5,12 +5,14 @@
 //!
 //! # Example
 //!
-//! ```rust
+//! ```rust, ignore
```

Comment:
It makes cargo test ignore this example as a test. It would otherwise fail because of the file access.